CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog
 
 ### Added
 
+* Added support for bgp, bgp_af, bgp_neighbor, bgp_neighbor_af
```

Comment:
For the release, wondering if it might be more clear to add all of the new supported features as their own section separate from the general "Added functonality" section. So something like:

### New feature support
* BGP (bgp, bgp_af, bgp_neighbor, bgp_neighbor_af)
* Radius (radius_global, radius_server)
* NTP (ntp_config, ntp_server)
...

### Added Functionality

* Enabled Travis-CI...
* Added git hooks
* Added best practices guide